src/components/side_panel/chart/sunburst_chart/sunburst_chart_design_panel.ts

```diff
@@ -37,12 +34,7 @@ export class SunburstChartDesignPanel extends Component<Props, SpreadsheetChildE
     ChartLegend,
     PieHoleSize,
   };
-  static props = {
-    chartId: String,
-    definition: Object,
-    updateChart: Function,
-    canUpdateChart: { type: Function, optional: true },
```

Comment:
pure curiositym do you know why it was set as optional in the first place? does not seem to match the TS interface